cluster_management/src/main/java/com/pinterest/rocksplicator/Participant.java

```diff
@@ -187,13 +191,20 @@ public static void disconnectHelixManager() throws Exception {
     }
   }
 
+  public HelixManager getHelixManager() {
+    return helixManager;
+  }
+
 
-  private Participant(String zkConnectString, String clusterName, String instanceName,
-                     String stateModelType, int port, String postUrl,
-                     boolean useS3Backup, String s3BucketName, boolean runSpectator) throws Exception {
+  public Participant(String zkConnectString, String clusterName, String instanceName,
+                      String stateModelType, int port, String postUrl,
+                      boolean useS3Backup, String s3BucketName, boolean runSpectator)
+      throws Exception {
```

Comment:
why these changes?

Reply:
I made the constructor to be public for the convenience of my unit testing of verifying registering to multiple modes. 
I removed the unit testing, seem to be unnecessary; thus, change back to "private".